packages/api/src/handlers/fetchProjectInRoundStatsHandler.ts

```diff
@@ -31,20 +50,30 @@ export const fetchProjectInRoundStatsHandler = async (req: Request, res: Respons
     switch (strategyName) {
       case "LINEAR_QUADRATIC_FUNDING":
         // fetch votes
-        const votes = await linearQFFetchVotesForProjectInRound(chainId, votingStrategyId, projectId);
-
+        const votes = await linearQFFetchVotesForProjectInRound(
+          chainId,
+          votingStrategyId,
+          projectId
+        );
+  
         // fetch round stats
-        results =  await linearQFFetchRoundStats(chainId, votes, metadata);
+        results = await linearQFFetchRoundStats(chainId, votes, metadata);
         break;
       default:
         return handleResponse(res, 400, "error: unsupported voting strategy");
     }
-
   } catch (err) {
     // TODO: LOG ERROR TO SENTRY
     // console.error(err);
-    return handleResponse(res, 500, "error: something went wrong.");
+    // serialize javascript error to json
```

Comment:
I would say serialize it for sentry error !
Let's send a generic 500 error and not the serialize json error